siliconcompiler/schema/schema_cfg.py

```diff
@@ -2124,6 +2150,17 @@ def schema_option(cfg):
             schelp="""
             Target PDK used during compilation.""")
 
+    scparam(cfg, ['option','uselambda'],
```

Comment:
May be a little ambiguous, especially if the 'lambda' value might end up being a sort of coarse measurement?